pantab/newsfragments/48.misc

```diff
@@ -0,0 +1 @@
+Build includes have been consolidated into the main setup script.
```

Comment:
You can remove this file - should only need notes for "user-facing" changes which this is not

Reply:
🤦‍♂ nuked it.